app/helpers/date_helper.rb

```diff
@@ -90,8 +90,11 @@ def time_select(f, field, options_tag = {}, html_options = {})
 
   def time_select24(f, field, options = {})
     options.reverse_merge! hours: (0..23)
-    output =  f.select(:"#{field}_hour", options[:hours].to_a)
-    output += f.select(:"#{field}_min", minute_options(options[:minute_step]))
+    hours = options.delete(:hours).to_a
+    minute_step = options.delete(:minute_step)
+
+    output = f.select(:"#{field}_hour", hours, {}, options)
+    output += f.select(:"#{field}_min", minute_options(minute_step), {}, options)
```

Comment:
Why are these changes needed? Was it a problem to use `options` as it were?

Reply:
This was needed to be able to disable the input with a `disabled:` argument, but I ended up not using it so I'll revert this change